src/utilities/indexer/queryFromIndexer.ts

```diff
@@ -0,0 +1,137 @@
+import { got } from 'got';
+
+import { configuration } from '../configuration';
+import { logger } from '../logger';
+import { sleep } from '../sleep';
+
+import { writeQuery } from './writeQuery';
+
+const { indexer } = configuration;
+
+const QUERY_INTERVAL_MS = 1000;
+export const QUERY_SIZE = 50;
+
+/** Example Query. */
+const queryBlocks = `
+  query {
+    blocks(orderBy: TIME_STAMP_DESC, first: 3) {
+    totalCount
+      nodes {
+        id
+        timeStamp
+        hash
+      }
+    }
+  }
+`;
+
+export interface FetchedData {
+  data: Record<
+    string,
+    {
+      totalCount?: number;
+      nodes?: Array<Record<string, unknown>>;
+    }
+  >;
+}
+
+export async function queryFromIndexer(query: string = queryBlocks) {
```

Comment:
I know I don't need to specify the type. But I like to be explicit.  Just like you prefer semantic syntax. 

I was using the "queryBlocks" at the start, to build my setup. 

Now I left it there as an example and set it as default for it to be used. 

I think it could be helpful while integrating the other apps.

Reply:
If it bothers to much I could deleted it and hide my tears.

Reply:
Succumbed here: [fix: remove default query](https://github.com/galaniprojects/ctypehub/commit/f7b1bef7b6ecf7669b618fb927e56d296cc6347e)